go/libraries/doltcore/sqle/index/prolly_fields.go

```diff
@@ -118,6 +118,8 @@ func GetField(ctx context.Context, td val.TupleDesc, i int, tup val.Tuple, ns tr
 		}
 	case val.CommitAddrEnc:
 		v, ok = td.GetCommitAddr(i, tup)
+	case val.CellEnc:
+		v, ok = td.GetCell(i, tup)
```

Comment:
We should throw an error here. We never want to be returning `Cell`s up to the SQL layer, they are only meaning full at the storage/indexing layer